dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -1291,17 +1291,30 @@ def test_integrated_merge_diff_options(self):
                                     'added together.'):
             profile1 + profile2
 
-    def test_clean_data_and_get_base_stats(self):
+    def test_clean_data_and_get_base_stats(self, *mocks):
         data = pd.Series([1, None, 3, 4, None, 6],
                          index=['a', 'b', 'c', 'd', 'e', 'f'])
 
         # validate that if sliced data, still functional
         # previously `iloc` was used at:
         # `df_series = df_series.loc[sorted(true_sample_list)]`
         # which caused errors
+
+        #Tests with default null values set
+        profiler = mock.Mock(spec=StructuredProfiler)
+        profiler._null_values = {
+            "": 0,
+            "nan": re.IGNORECASE,
+            "none": re.IGNORECASE,
+            "null": re.IGNORECASE,
+            "  *": 0,
+            "--*": 0,
+            "__*": 0,
+        }
+
         test_utils.set_seed(seed=0)
         df_series, base_stats = \
-            StructuredColProfiler.clean_data_and_get_base_stats(
+            StructuredColProfiler.clean_data_and_get_base_stats(profiler,
```

Comment:
Wouldn't it be better to do `profiler.clean_data_and_get_base_stats`? I've never seen class methods called this way on non-static methods. They're listed below too, I don't know if this is normal and I'm just unfamiliar or if this should be changed. Thoughts @JGSweets?

Reply:
It's because we are using a mock on 1304.